src/EnlistmentInfo.targets

```diff
@@ -8,10 +8,11 @@
     <DocumentationFile Condition=" !$(IsTestProject) ">$(OutputPath)$(AssemblyName).xml</DocumentationFile>
     <DefineConstants Condition=" '$(TargetFrameworkProfile)' == '' ">$(DefineConstants);DESKTOP</DefineConstants>
   </PropertyGroup>
-
+  
   <ItemGroup>
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)stylecop.json"/>
   </ItemGroup>
 
   <Import Project="NuProj.Extra.targets" Condition=" '$(MSBuildProjectExtension)' == '.nuproj' "/>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), ExportMethods.targets))\ExportMethods.targets" Condition=" $(MSBuildProjectName.Contains('Desktop')) OR $(MSBuildProjectName) == 'BCrypt' "/>
```

Comment:
What about "All projects but NuGet or Test", use the same shared directory and the project name to create the txt files (Ex:. Kernel32.Desktop.pinvokes.txt, Kernel32.pinvokes.txt).
Then as you pointed:
>address the question of how to pick the right files to compare later